engine/src/witness/eth.rs

```diff
@@ -162,8 +156,8 @@ where
 			usdc_contract_address,
 		)
 		.await?
-		.continuous("USDCDeposits".to_string(), db.clone())
-		.logging("USDCDeposits")
+		.continuous("EthereumUSDCDeposits".to_string(), db.clone())
+		.logging("EthereumUSDCDeposits")
```

Comment:
same here, no need to prepend with `Ethereum` for the logging adapter, it's already included